src/BEditaClient.php

```diff
@@ -412,6 +412,32 @@ public function createMediaFromStream($streamId, $type, $body) : array
         return $this->getObject($data['id'], $data['type']);
     }
 
+    /**
+     * Thumbnail request using `GET /media/thumbs` endpoint
+     *
+     *  Usage:
+     *          thumbs(123) => `GET /media/thumbs/123`
+     *          thumbs(123, ['preset' => 'glide']) => `GET /media/thumbs/123&preset=glide`
+     *          thumbs(null, ['ids' => '123,124,125']) => `GET /media/thumbs?ids=123,124,125`
+     *          thumbs(null, ['ids' => '123,124,125', 'preset' => 'async']) => `GET /media/thumbs?ids=123,124,125&preset=async`
```

Comment:
add an example of explicit width/height parameters in query string - this request may not be available